studio-docs/source/sharing-graphs.mdx

```diff
@@ -0,0 +1,504 @@
+    </div>
+  );
+}
+```
+
+</MultiCodeBlock>
+
+##### Providing a `graphRef`
+
+Notice above that we append some query parameters to our `iframe`'s URL. The most important of these is `graphRef`. This tells the Explorer which [public variant's](#public-graphs) schema to download to populate the Documentation panel and enable symbol completion.
+
+> **Always provide a `graphRef`!** If you don't, the embedded Explorer can't download your schema to enable features like symbol completion and documentation.
+
+Other supported query parameters (like `docsPanelState` used above) are listed in [Options](#options).
+
+#### 5. Run the example
+
+You can view and run a complete example that uses code very similar to the above on Codesandbox:
+
+<a href="https://codesandbox.io/s/github/apollographql/embeddable-explorer-demo-react/tree/main/?fontsize=14&hidenavigation=1&theme=dark">
+  <img alt="Edit embedded-explorer-demo" src="https://codesandbox.io/static/img/play-codesandbox.svg" />
+</a>
+
+The example includes support for subscriptions, which are omitted from this Setup section for clarity. The subscription-specific logic should be more straightforward after familiarizing yourself with the logic for queries and mutations.
+
+### Options
+
+When you embed the Explorer, you set options for it by including query parameters in the `src` URL of the `iframe`, like so:
+
+```jsx:title=EmbeddedExplorer.jsx
+return (
+  <iframe src={`https://explorer.embed.apollographql.com/?graphRef=${myGraphRef}&docsPanelState=closed`} />
+);
+```
+
+The following options are supported:
+
+<table class="field-table api-ref">
+  <thead>
+    <tr>
+      <th>Name /<br/>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr class="required">
+<td>
+
+##### `graphRef`
+
+`string`
+
+</td>
+<td>
+
+**Required.** The graph ref for the [public variant](#public-graphs) you want to use the embedded Explorer with. Has the format `graph-name@variant-name`.
+
+The Explorer fetches this variant's schema from Apollo Studio to populate its Documentation panel and enable symbol completion.
+
+If you omit this option, the Explorer _does_ still load, but it doesn't know which schema to fetch. This prevents the Explorer from providing critical features like documentation and symbol completion.
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `docsPanelState`
+
+`"open" | "closed"`
+
+</td>
+<td>
+
+If `open`, the Explorer's Documentation panel (the left column) is initially expanded. If `closed`, the panel is initially collapsed.
+
+The default value is `open`.
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `showHeadersAndEnvVars`
+
+`"true" | "false"`
+
+</td>
+<td>
+
+If `true`, the embedded Explorer includes the panels for [setting request headers and environment variables](./explorer/#authentication). If `false`, those panels are not present.
+
+The default value is `true`.
+
+</td>
+</tr>
+
+<tr>
+<td>
+
```

Comment:
```suggestion
##### `performOperationsInEmbed`

`"true" | "false"`

</td>
<td>

If `true`, when you submit an operation through the explorer, we will use your graph ref's endpoint to perform operations, instead of post messaging the network requests to the parent page. 

The default value is `false`.

</td>
</tr>
</tbody>
</table>
```

Reply:
This is something we are adding in! PR: https://github.com/mdg-private/studio-ui/pull/4904

Reply:
You have to be on a public graph with a url endpoint set that that to work, though.